docs/content/configuration.md

```diff
@@ -32,8 +32,15 @@ The service mode can be enabled by using the following annotation:
 maesh.containo.us/traffic-type: "http"
 ```
 
-This annotation can be set to either `http` or `tcp`, and will specify the mode for that service operation.
-If this annotation is not present, the mesh service will operate in the default mode specified in the static configuration.
+### Scheme
```

Comment:
Did we want to remove the description for the previous annotation?

Also, we should probably mention that scheme is only applicable if `traffic-type` is http